orcid-activemq/src/main/resources/message-listener.properties

```diff
@@ -70,4 +70,15 @@ org.orcid.message-listener.index.s3.search.max_elements=5
 org.orcid.message-listener.index.summaries.bucket_name=v2.0-summaries-dev
 org.orcid.message-listener.index.activities.bucket_name=v2.0-activities-dev
 org.orcid.message-listener.index.summaries.v3.bucket_name=v3.0-summaries-dev
-org.orcid.message-listener.index.activities.v3.bucket_name=v3.0-activities-dev
\ No newline at end of file
+org.orcid.message-listener.index.activities.v3.bucket_name=v3.0-activities-dev
+
+
+org.orcid.persistence.panoply.cleanup.production=false
```

Comment:
@Camelia-Orcid do we need this one here? org.orcid.persistence.panoply.cleanup.production=false